addons/account/views/report_invoice.xml

```diff
@@ -13,9 +13,8 @@
                         </div>
                     </t>
                 </t>
-                <div t-field="o.partner_id" t-options='{"widget": "contact", "fields": ["address", "name"], "no_marker": True}' groups="account.group_delivery_invoice_address"/>
                 <t t-set="address">
-                    <address t-field="o.partner_id" t-options='{"widget": "contact", "fields": ["address", "name"], "no_marker": True}' groups="!account.group_delivery_invoice_address"/>
+                    <address t-field="o.partner_id" t-options='{"widget": "contact", "fields": ["address", "name"], "no_marker": True}' groups="account.group_delivery_invoice_address"/>
```

Comment:
You should remove `groups="account.group_delivery_invoice_address"`.
Otherwise, no address will be displayed if `Customer Addresses` is deactivated in the settings.